subworkflows/nf-core/alignment_to_fastq.nf

```diff
@@ -15,6 +15,7 @@ workflow ALIGNMENT_TO_FASTQ {
     take:
     input // channel: [meta, alignment (BAM or CRAM), index (optional)]
     fasta // optional: reference file if CRAM format and reference not in header
+    fasta_fai
```

Comment:
Actually, we don't need to pass either fasta nor fai since we have bam files at this step, right?

Reply:
ok, never mind, I can see that we pass `[]` to this subworkflow